src/openrct2/interface/Theme.cpp

```diff
@@ -160,6 +160,7 @@ WindowThemeDesc WindowThemeDescriptors[] =
     { THEME_WC(WC_TITLE_OPTIONS),                  STR_THEMES_WINDOW_TITLE_MENU_OPTIONS,       COLOURS_3(TRANSLUCENT(COLOUR_DARK_GREEN),  TRANSLUCENT(COLOUR_DARK_GREEN),  TRANSLUCENT(COLOUR_DARK_GREEN)                     ) },
     { THEME_WC(WC_LAND_RIGHTS),                    STR_THEMES_WINDOW_LAND_RIGHTS,       COLOURS_3(COLOUR_DARK_YELLOW,              COLOUR_DARK_YELLOW,              COLOUR_DARK_YELLOW                                 ) },
     { THEME_WC(WC_THEMES),                         STR_THEMES_WINDOW_THEMES,       COLOURS_3(COLOUR_GREY,                     COLOUR_DARK_GREEN,               COLOUR_DARK_GREEN                                  ) },
+    { THEME_WC(WC_FOOTPATH_SCENERY),               STR_THEMES_WINDOW_FOOTPATH_SCENERY,    COLOURS_3(COLOUR_GREY,                     COLOUR_LIGHT_BLUE,               COLOUR_LIGHT_BLUE                                  ) },
```

Comment:
Setting only one colour makes the widgets "black", which makes the text nearly impossible to read. I could change the text colour to white, but that looks a bit odd.
![pathscenery_black](https://cloud.githubusercontent.com/assets/8963459/21680468/4365f42a-d34a-11e6-97a5-50667059b8a4.png)